kustomize/base/single_cluster/headless/config/admin/server.yaml

```diff
@@ -10,6 +10,23 @@ server:
       - "*"
     allowedHeaders:
       - "Content-Type"
+      - "flyte-authorization"
+auth:
+  appAuth:
+    thirdPartyConfig:
+      flyteClient:
+        clientId: flytectl
+        redirectUri: https://localhost:53593/callback
+        scopes:
+          - offline
+          - all
+  userAuth:
+    openId:
+      baseUrl: https://accounts.google.com
```

Comment:
One issue with the base url is that https://accounts.google.com/.well-known/oauth-authorization-server does not exist.

Reply:
If you put `https://accounts.google.com` under `appAuth -> baseUrl` then it won't work because Google doesn't offer An authorization server for personal use.. (outside of GCP).

if you put it (as in this file) under `userAuth -> openId -> baseUrl`, we won't try to hit that well known url, we will, instead, hit https://accounts.google.com/.well-known/openid-configuration which is available...